extensions-core/lookups-cached-global/src/main/java/org/apache/druid/server/lookup/namespace/JdbcCacheGenerator.java

```diff
@@ -52,6 +56,14 @@
   private final ConcurrentMap<CacheScheduler.EntryImpl<JdbcExtractionNamespace>, DBI> dbiCache =
       new ConcurrentHashMap<>();
 
+  PeriodFormatter formatter = new PeriodFormatterBuilder()
```

Comment:
This looks like `PeriodFormat.getDefault()` (which doesn't need to be re-cached in a field, could be used directly).

Reply:
done